.gitpod.yml

```diff
@@ -37,7 +37,6 @@ tasks:
       nvm use v16.4.0
       npm i
       make run
-    openMode: split-right
```

Comment:
this is deleted because of this `Dev` terminal is not in focus even though it is in last place. (P.S. I also tried to put it in the first place but it didnt work. I will create an issue on Gitpod)